unattended_scripts/open-distro/unattended-installation/distributed/wazuh-server-installation.sh

```diff
@@ -11,7 +11,8 @@
 ## Check if system is based on yum or apt-get
 char="."
 debug='> /dev/null 2>&1'
-WAZUH_VER="4.2.2"
+WAZUH_MAJOR="4.3"
```

Comment:
This have to keeps stable version.

Reply:
Updated to 4.4, this should have the current version